src/app/tests/suites/certification/Test_TC_MF_1_4.yaml

```diff
@@ -16,398 +16,145 @@ name: 22.1.4. [TC-MF-1.4] Node Behavior using BCM [DUT - Commissionee]
-          [1635865583.366502][3951:3956] CHIP:DMG: {
-          [1635865583.366570][3951:3956] CHIP:DMG: 0x0 = 2,
-          [1635865583.366682][3951:3956] CHIP:DMG: 0x1 = [
-          [1635865583.366814][3951:3956] CHIP:DMG: 0x4, 0x5b, 0x93, 0x38, 0xab,
-          0x50, 0x6, 0x85, 0xc0, 0x50, 0xfe, 0x7d, 0x6c, 0xeb, 0x9d, 0xbd, 0x55,
-          0x90, 0x5e, 0x4f, 0x54, 0xed, 0x94, 0x21, 0xc0, 0x8d, 0x9d, 0x4f,
-          0x60, 0x48, 0xdb, 0x38, 0x4a, 0xf4, 0xe2, 0xa3, 0xf9, 0xd0, 0x55,
-          0xd5, 0x63, 0xd0, [1635865583.366901][3951:3956] CHIP:DMG: ]
-          [1635865583.366970][3951:3956] CHIP:DMG: 0x2 = 45896,
-          [1635865583.367068][3951:3956] CHIP:DMG: 0x3 = 0,
-          [1635865583.367123][3951:3956] CHIP:DMG: 0x4 = 2,
-          [1635865583.367272][3951:3956] CHIP:DMG: 0x5 = '',
-          [1635865583.367358][3951:3956] CHIP:DMG: },
-          [1635865583.367499][3951:3956] CHIP:DMG: ],
-          [1635865583.367581][3951:3956] CHIP:DMG: DataElementVersion = 0x0,
-          [1635865583.367653][3951:3956] CHIP:DMG: },
-          [1635865583.367720][3951:3956] CHIP:DMG:
-          [1635865583.367764][3951:3956] CHIP:DMG: ],
-          [1635865583.367829][3951:3956] CHIP:DMG:
-          [1635865583.367871][3951:3956] CHIP:DMG: }
-          [1635865583.368097][3951:3956] CHIP:ZCL: ReadAttributesResponse:
-          [1635865583.368144][3951:3956] CHIP:ZCL: ClusterId: 0x0000_003E
-          [1635865583.368188][3951:3956] CHIP:ZCL: attributeId: 0x0000_0001
-          [1635865583.368228][3951:3956] CHIP:ZCL: status: Success (0x0000)
-          [1635865583.368319][3951:3956] CHIP:ZCL: attribute TLV Type: 0x16
-          [1635865583.368389][3951:3956] CHIP:TOO:
-          OnOperationalCredentialsFabricsListListAttributeResponse: 2 entries
-          [1635865583.368513][3951:3956] CHIP:TOO: FabricDescriptor[1]:
-          [1635865583.368555][3951:3956] CHIP:TOO: fabricIndex: 1
-          [1635865583.368593][3951:3956] CHIP:ZCL: RootPublicKey: 65
-          [1635865583.368630][3951:3956] CHIP:TOO: vendorId: 30024
-          [1635865583.368665][3951:3956] CHIP:TOO: fabricId: 0
-          [1635865583.368696][3951:3956] CHIP:TOO: nodeId: 1
-          [1635865583.368729][3951:3956] CHIP:ZCL: Label:
-          [1635865583.368777][3951:3956] CHIP:TOO: FabricDescriptor[2]:
-          [1635865583.368816][3951:3956] CHIP:TOO: fabricIndex: 2
-          [1635865583.368862][3951:3956] CHIP:ZCL: RootPublicKey: 65
-          [1635865583.368895][3951:3956] CHIP:TOO: vendorId: 45896
-          [1635865583.368971][3951:3956] CHIP:TOO: fabricId: 0
-          [1635865583.369011][3951:3956] CHIP:TOO: nodeId: 2"
-      disabled: true
+    #Issue 16094 Need to check length of list and verify its content
+    - label: "4 Query fabrics list"
+      command: "readAttribute"
+      cluster: "Operational Credentials"
+      attribute: "Fabrics"
+      response:
+          value: [{ Label: "", nodeId: [nodeId, nodeId2] }]
```

Comment:
This does not make sense.  `nodeId` is not a list.

Reply:
Updated

Reply:
@bzbarsky-apple We had a discussion with raju and he was asking us to use "non-fabric-filtered read" for this test step, Its not clear to us how this can be done in YAML, is it just that the identifier needs to be removed ?

Reply:
`fabricFiltered: false`

See also `TestClusterMultiFabric.yaml` for some examples.